Authentication/Login/View/LoginViewDelegate.swift

```diff
@@ -25,13 +24,13 @@ public protocol LoginViewDelegate {
      
         - Parameter loginView: view to configure.
      */
-    func configureView(loginView: LoginViewType)
+    func configureView(_ loginView: LoginViewType)
```

Comment:
I prefer `public func configureLoginView(_ loginView: LoginViewType)`, because it falls more in line with naming conventions.